src/store/Application.js

```diff
@@ -1,48 +1,49 @@
-import { isValidJWT } from "@/utils/helpers";
-import store from "@/store";
-
-const state = {};
-
-const mutations = {};
-
-const actions = {
-    getGlobalStateData({ dispatch }) {
-        // RETURN IF NO COOKIES FOUND SENT ERROR (CATCH) !Cookies.get('userID'
-        if (!Cookies.get("token") || !isValidJWT(Cookies.get("token"))) {
-            return new Promise((resolve, reject) => {
-                reject("ERROR");
-            });
-        }
-
-        return Promise.all([
-            dispatch("User/getData", null, { root: true }),
-            dispatch("Company/getData", null, { root: true })
-        ]).then(response => {
-            dispatch("setGlobalData", { userData: response[0].data, companies: response[1].data });
-        })
+import { isValidJWT } from "@/utils/helpers";
+import store from "../store/index";
```

Comment:
No need to use `index`.
```suggestion
import store from "../store";
```